docs/README.md

```diff
@@ -62,27 +62,38 @@ The BioCro development team may make changes to the skeleton module library in
 the future; every effort will be made to limit the frequency of these updates,
 but they will nevertheless occur. When there is an update to this repository,
 you can update your module library by taking the following steps:
-1. Make a new "development" branch for your repository and publish it to GitHub.
-2. On the GitHub interface for your repository, switch to the development branch
-   and click the `Sync fork` button to pull in the latest changes from
-   `skelBML`.
-3. Pull the changes to your local machine.
-4. Check `NEWS_skeleton.md` to see if the new version of the skeleton module
+1. Make a new "development" branch for your repository (based on the main
+   branch).
+2. The GitHub web interface has a "Sync fork" button, but it cannot be used if
+   there are any conflicts between your repository and `skelBML`. Unfortunately,
+   this will almost always be the case when updating your repository. Instead,
+   you can sync your branch to `skelBML` using command-line git as follows
+   (assuming you have checked out the new development branch on your local
+   machine):
```

Comment:
Since I think it might be a little too easy to miss that your working copy must be on the development branch before running the commands, I'd prefer to see something like the following.  (While writing this, I also realized that a user following these instructions may not have set up the `upstream` remote.  Perhaps GitHub Desktop does this automatically, but a command-line git user "cloning the repository as usual" will only get the `origin` remote, pointing to their fork.)

---

__Instead, you can sync your branch to `skelBML` using git from the command line as follows: First, be sure the new development branch is checked out  in your working copy and that you have a remote named `upstream` that points to the `biocro/skelBML` GitHub repository (_this_ repository).  Then, run the commands__

    git fetch upstream
    git merge upstream/main